src/openrct2/core/String.cpp

```diff
@@ -706,6 +706,7 @@ namespace String
     std::string ToUpper(const std::string_view& src)
     {
 #ifdef _WIN32
+#    if _WIN32_WINNT >= 0x0600
```

Comment:
Is it a standard in the whole project to have macros with the content indented but not the `#`? I see it in the includes and though I know it doesn't make a difference, it still bugs me

Reply:
I think it bugs quite a few people. There needs to be a change to our clang-format config and then a bulk format edit to the code base though.